src/AzureDBExperiences.ts

```diff
@@ -81,15 +81,24 @@ export function getExperienceQuickPicks(): IAzureQuickPickItem<Experience>[] {
     return experiencesArray.map(exp => getExperienceQuickPick(exp.api));
 }
 
-export function getCosmosExperienceQuickPicks(): IAzureQuickPickItem<Experience>[] {
-    return cosmosExperiencesArray.map(exp => getExperienceQuickPick(exp.api));
+export function getCosmosExperienceQuickPicks(attached?: boolean): IAzureQuickPickItem<Experience>[] {
+    if (attached) {
+        return cosmosExperiencesArray.map(exp => getExperienceQuickPickforAttached(exp.api));
+    } else {
+        return cosmosExperiencesArray.map(exp => getExperienceQuickPick(exp.api));
+    }
 }
 
 export function getExperienceQuickPick(api: API): IAzureQuickPickItem<Experience> {
     const exp = getExperienceFromApi(api);
     return { label: exp.longName, description: exp.description, data: exp };
 }
 
+export function getExperienceQuickPickforAttached(api: API): IAzureQuickPickItem<Experience> {
```

Comment:
nit: to preserve camel casing

```suggestion
export function getExperienceQuickPickForAttached(api: API): IAzureQuickPickItem<Experience> {
```